prime-router/src/main/kotlin/common/PhoneUtilities.kt

```diff
@@ -0,0 +1,59 @@
+package gov.cdc.prime.router.common
```

Comment:
i like this being its own file. some of the other convertion code may benefit from being like this too